TACC/TACC-Docs
# fix: tup-727 tip admonition renders wrong

## Overview

The "tip" admonition was rendering incorrectly because of a bad selector in a Core-Styles stylesheet.

## Related

- [TUP-727](https://tacc-main.atlassian.net/browse/TUP-727)

## Changes

- **updated** Core-Styles

## Testing

1. Open http://127.0.0.1:8000/hpc/stampede3/#sjobs-monitoring-queuestatus.
2. Verify _blue box with light bulb_ around each of two blocks.

## UI

| before | after |
| - | - |
| <img width="1194" alt="TUP-727 before" src="https://github.com/user-attachments/assets/99891541-a565-48e4-808f-f42c6b519b7d"> | <img width="1194" alt="TUP-727 after" src="https://github.com/user-attachments/assets/9f144863-ec12-4167-a2f5-c4ee8721a939"> |
